company_eori_number/README.rst

```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+===================
+company_eori_number
+===================
+
+This module was written to extend the functionality of the company form to support EORI number (Economic Operators Registration and Identification number) .
+
+
+Installation
+============
+
+To install this module, you need to:
+
+
+#. Install the module.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. No need for configuration
```

Comment:
If something does not apply, remove the section completely (ex. remove Configuration)